docs/pages/reference/helm-reference/teleport-kube-agent.mdx

```diff
@@ -106,6 +106,90 @@ This parameter is not mandatory to preserve backwards compatibility with older c
 
 If you specify a role here, you may also need to specify some other settings which are detailed in this reference.
 
+## `updater`
+
+`updater` controls whether the Kube Agent Updater should be deployed alongside
+the `teleport-kube-agent`. The updater fetches the target version, validates the
+image signature, and updates the teleport deployment.
+
+All kubernetes-specific fields such as `tolerations`, `affinity`, `nodeSelector`,
+... default to the agent values. However, they can be overridden from the
+`udpater` object. For example:
+
+```yaml
+# the agent pod requests 1cpu and 2 GiB of memory. It also has a memory limit.
+resources:
+  requests:
+    cpu: "1"
+    memory: "2Gi"
+  limits:
+    memory: "2Gi"
+
+# the updater pod requests 0.5 cpu and 512MiB of memory. The memory limit has also been unset.
+updater:
+  resources:
+    requests:
+      cpu: "0.5"
+      memory: "512Mi"
+    limits: ~
+```
+
+Other updater-specific values that can be defined in `updater` are described
+below:
+
+### `updater.enabled`
+
+Enables the Kube Agent Updater and deploys it alongside the Teleport Agent.
+You must enable this when:
+- using Teleport Cloud and you tenant is enrolled into automatic updates.
+  (You can check this through the web UI, choose "add a new node" of type
+  Kubernetes, and check if the value is turned on.)
+- using Teleport on-prem and you maintain your own version server.
```

Comment:
Can we provide an example of such server?
Even if it only returns static values
A simple `net/http` based program should be enough

Reply:
👍 , I have an issue to write documentation on how to host your own version server. Will address this in an other PR.